docs/version0.33.md

```diff
@@ -36,6 +36,8 @@ The following new features and notable changes since version 0.30.0 are included
 
 OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
 
+Windows&reg; builds for Java&trade; 8 are now compiled with Microsoft&reg; Visual Studio 2017. The Visual Studio redistributable files included with the build are updated to match.
```

Comment:
Should this refer to "OpenJDK 8" instead of "Java 8"?

Reply:
It doesn't hurt so I've changed it. I don't know that we've been consistent about it one way or the other in these release notes, I copied the text from 0.24.